test/Filter/MethodMatchFilterTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2018 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+use PHPUnit\Framework\TestCase;
+use Zend\Hydrator\Filter\MethodMatchFilter;
+
+/**
```

Comment:
Please remove the entire class comment, because it is not needed and wrong.

Reply:
done